drift-client/src/main/java/io/airlift/drift/client/guice/DriftClientBinder.java

```diff
@@ -81,7 +81,10 @@ private DriftClientBinder(Binder binder)
 
     public <T> DriftClientBindingBuilder bindDriftClient(Class<T> clientInterface, Class<? extends Annotation> annotationType)
     {
-        String configPrefix = getServiceName(clientInterface) + "." + annotationType.getSimpleName();
+        String configPrefix = getServiceName(clientInterface);
+        if (annotationType != DefaultClient.class) {
+            configPrefix += "." + annotationType.getSimpleName();
```

Comment:
Not related to this commit, but didn't we want to reverse this so the annotation name comes first?